example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -746,36 +765,38 @@ describe('When there are granule differences and granule reconciliation is run',
       const exists = await fileExists(parsed.Bucket, parsed.Key);
       expect(exists).toBeFalse();
 
-      const response = await reconciliationReportsApi.getReconciliationReport({
-        prefix: config.stackName,
-        name: reportRecord.name,
-      });
-
-      expect(response.statusCode).toBe(404);
-      expect(JSON.parse(response.body).message).toBe(`No record found for ${reportRecord.name}`);
+      let responseError;
+      try {
+        await reconciliationReportsApi.getReconciliationReport({
+          prefix: config.stackName,
+          name: reportRecord.name,
+        });
+      } catch (error) {
+        responseError = error;
+      }
+      expect(responseError.statusCode).toBe(404);
+      expect(JSON.parse(responseError.apiMessage).message).toBe(`No record found for ${reportRecord.name}`);
     });
   });
 
   afterAll(async () => {
-    console.log(`update granule files back ${publishedGranuleId}`);
+    const activeCollectionId = constructCollectionId(extraCumulusCollection.name, extraCumulusCollection.version);
```

Comment:
A good portion of this cleanup wasn't working as intended, particularly the extra collection cleanup.   

Given the nature of that cleanup, and this test implementing an API-driven top-down collection deletion seems more maintainable/DRY/etc than the prior approach, however feedback is welcome.